fetch.bs

```diff
@@ -4506,6 +4506,9 @@ steps:
 
  <li><p>If <var>recursive</var> is false, then run the remaining steps <a>in parallel</a>.
 
+ <li><p>Let |raceResponse| be the result of <a for=/>lookup race response</a> given <var>request</var>.
+ <li><p>If |raceResponse| is <var>response</var>, return |raceResponse|.
```

Comment:
_response_ is null at this point, so I doubt this check is what you mean.

Reply:
I meant |response| is a [response](https://fetch.spec.whatwg.org/#concept-response). Updated.